### DISCUSSION TITLE
Default theme does not load OS theme colours properly

### DISCUSSION THREAD

**Discussion by Nihlus** [dbeaver/dbeaver#621] [Q&A] [answered]
When using a dark GTK theme on Linux, certain elements in the UI are too bright or have badly matched colours, making it difficult to see.

![image](https://cloud.githubusercontent.com/assets/2340428/17051190/93fabfc8-4ff7-11e6-8615-0f00a4ac57f6.png)

This is most apparent with the GTK theme enabled in settings, but is an issue with all themes. I believe the issue lies in hardcoded colours.

**Comment by slestak** [comment]
See #598

**Comment by serge-rider** [comment]
You can configure DBeaver specific colors in Preferences->General->Appearance->Colors and Fonts.

**Comment by andrei-gheorghiu** [comment, marked as answer]
Until someone with enough design skills spends some time to correct each color and export the result, a quick fix is to go to 
Window > Preferences > General > Appearance
 1. Check "Enable theming"
 2. Theme: Dark

It's not as slick as GTK (or similar), but it makes the data readable.
